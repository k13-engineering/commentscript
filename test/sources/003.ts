import { thisLineNumber } from "../common/line-nr.js";

interface PublicInterface {
};

const abc = () => {
  interface IScopedInterface {
  };
};

const test = async () => {
  const lineNr = await thisLineNumber();
  const expectedLineNr = 12;
  if (lineNr !== expectedLineNr) {
    throw Error(`line number is not ${expectedLineNr} but ${lineNr}`);
  }
};

// function with same name than scoped interface
const IScopedInterface = () => {

};

export {
  PublicInterface,
  abc,
  test,
  IScopedInterface
};
