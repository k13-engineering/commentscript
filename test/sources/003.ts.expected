import { thisLineNumber } from "../common/line-nr.js";

/*interface PublicInterface {
}*/ const PublicInterface = {};

const abc = () => {
  /*interface IScopedInterface {
  }*/ const IScopedInterface = {};
};

const test = async () => {
  const lineNr = await thisLineNumber();
  const expectedLineNr = 12;
  if (lineNr !== expectedLineNr) {
    throw Error(`line number is not ${expectedLineNr} but ${lineNr}`);
  }
};

// function with same name than scoped interface
const IScopedInterface = () => {

};

export {
  PublicInterface,
  abc,
  test,
  IScopedInterface
};
